Example/Unit Tests/AnimationSnapshotTests.swift

```diff
@@ -23,6 +23,8 @@ final class AnimationSnapshotTests: FBSnapshotTestCase {
     override func setUp() {
         super.setUp()
 
+        fileNameOptions = [.OS, .screenSize, .screenScale]
```

Comment:
How does this get used?

Reply:
This is how the snapshot framework determines what the file name should be for the reference images. So if you look at one of the reference images:

```
testAnimationWithExecutionBlocksSnapshotGIF_11_4_375x812@3x.gif
```

`testAnimationWithExecutionBlocksSnapshotGIF` is the test name
`11_4` is the OS version
`375x812` is the screen size
`@3x` is the screen scale